### ISSUE TITLE
Very poor contrast with light grey text

### ISSUE THREAD

**Issue by Keats** [rust-lang/crates.io#1286]
Look at this page for example: https://crates.io/categories

`Displaying 1-46 of 46 total results `: the light grey has a ratio of 3.43:1 compared to the background, failing the https://www.w3.org/TR/WCAG20/

The rest of the grey text (num of crates in each category and description) also fails with a ration of 3.69:1

Quick reminder of the levels from the spec:

> WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. Level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text.

This is small text so I guess even 4.5:1 would be too light? 

As an aside, the spacing on those categories feels wrong. Adding a `margin-bottom: 0.5rem` to the `info` class let the header breath a bit.

**Comment by sgrif** [comment]
It sounds like you have some very specific ideas of what should be done here. I'd love to get a PR for those changes.

**Comment by Turbo87** [comment]
this appears to be a duplicate of, or at least partly covered by https://github.com/rust-lang/crates.io/issues/1233, so I'll close this issue here as we already have one that tracks the color contrast improvements.
